packages/gatsby/src/utils/page-data.ts

```diff
@@ -136,7 +136,19 @@ export async function flush(): Promise<void> {
 
   const { pagePaths } = pendingPageDataWrites
 
-  const pagesToWrite = pagePaths.values()
+  const pagesToWrite = Array.from(pagePaths.values())
+
+  let writePageDataActivity
+  // Don't create the activity in tests as it changes
+  // behavior for the get-page-data tests.
```

Comment:
having an activity changed the number of `setTimeouts` in https://github.com/gatsbyjs/gatsby/blob/589051f181e9b94f57f60c1eb3a5c4beb7425b82/packages/gatsby/src/utils/__tests__/get-page-data.ts#L207 I'm guessing the `.tick()` did that? Anyways, didn't seem worth changing so just turned off the activity during tests.

Reply:
Please mock the createProgress reporter so we still test these pieces.

Reply:
👍